android/app/src/main/java/com/yagubogu/ui/badge/component/BadgeBottomSheet.kt

```diff
@@ -0,0 +1,153 @@
+                thickness = 0.4.dp,
+                color = Gray300,
+                modifier = Modifier.padding(vertical = 20.dp),
+            )
+            Text(
+                text = badge.description,
+                style = PretendardMedium,
+                fontSize = 14.sp,
+                textAlign = TextAlign.Center,
+            )
+            Spacer(modifier = Modifier.height(30.dp))
+            if (badge.isAcquired) {
+                Button(
+                    onClick = onRegisterClick,
+                    colors =
+                        ButtonColors(
+                            containerColor = Primary500,
+                            contentColor = White,
+                            disabledContainerColor = Primary500,
+                            disabledContentColor = White,
+                        ),
+                    shape = RoundedCornerShape(12.dp),
+                    contentPadding = PaddingValues(vertical = 16.dp),
+                    modifier = Modifier.fillMaxWidth(),
+                ) {
+                    Text(
+                        text = stringResource(R.string.badge_register_main_badge),
+                        style = PretendardBold16,
+                    )
+                }
+                Text(
+                    text =
+                        stringResource(
+                            R.string.badge_achieved_date,
+                            badge.achievedAt.format(DateFormatter.yyyyMMdd),
+                        ),
+                    style = PretendardRegular12,
+                    color = Gray500,
+                    modifier = Modifier.padding(top = 12.dp),
+                )
+            } else {
+                LinearProgressIndicator(
+                    progress = { badge.progressRate.toFloat() / 100 },
+                    color = Primary500,
+                    trackColor = Gray300,
+                    gapSize = (-12).dp,
+                    drawStopIndicator = {},
+                    modifier =
+                        Modifier
+                            .fillMaxWidth()
+                            .height(12.dp),
+                )
+                Text(
+                    text =
+                        stringResource(
+                            R.string.badge_progress_rate,
+                            badge.progressRate,
+                        ),
+                    style = PretendardRegular12,
+                    color = Gray500,
+                    modifier = Modifier.padding(top = 12.dp),
+                )
+            }
+
+            Spacer(modifier = Modifier.height(20.dp))
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Preview
+@Composable
+private fun BadgeBottomSheetPreview() {
+    BadgeBottomSheet(
+        badge = BADGE_NOT_ACQUIRED_FIXTURE,
+        onRegisterClick = {},
+        onDismiss = {},
+        sheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true),
```

Comment:
```suggestion
        sheetState = rememberStandardBottomSheetState(),
```

프리뷰에서 바로 볼 수 있게 이건 어떤가용